include/mbgl/util/geo.hpp

```diff
@@ -103,14 +107,18 @@ class LatLngBounds {
 
     // Return a bounds that may serve as the identity element for the extend operation.
     static LatLngBounds empty() {
-        LatLngBounds bounds = world();
+        LatLngBounds bounds = LatLngBounds::world();
         std::swap(bounds.sw, bounds.ne);
         return bounds;
     }
 
     // Constructs a LatLngBounds object with the tile's exact boundaries.
     LatLngBounds(const CanonicalTileID&);
 
+    explicit operator bool() const {
+        return (sw.latitude <= ne.latitude) && (sw.longitude <= ne.longitude);
```

Comment:
This would be consistent with the GeoJSON specification, but inconsistent with GeoJSONVT’s behavior of allowing a single feature or annotation to span the antimeridian (#6088), correct? If one were to [calculate the bounding box of an annotation](https://github.com/mapbox/mapbox-gl-native/blob/62ea1f21858c69f6921c775ba7a3de201f0514d8/platform/darwin/src/MGLMultiPoint.mm#L164-L167), would this operator end up evaluating to `false`? Would it still be possible to [fit the bounds to the annotation](https://github.com/mapbox/mapbox-gl-native/blob/6c92199a16a9fc6648244547a183bc80da7a56d7/platform/ios/src/MGLMapView.mm#L3960-L3972)?